tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -1631,34 +1631,15 @@ public function test_wp_script_add_data_with_data_key() {
 	/**
 	 * Testing `wp_script_add_data` with the conditional key.
 	 *
+	 * @since 6.9.0 Conditional comments should now return an empty string.
+	 *
 	 * @ticket 16024
 	 */
 	public function test_wp_script_add_data_with_conditional_key() {
 		// Enqueue and add conditional comments.
 		wp_enqueue_script( 'test-only-conditional', 'example.com', array(), null );
 		wp_script_add_data( 'test-only-conditional', 'conditional', 'gt IE 7' );
```

Comment:
These could assert the false return that was added here:


```suggestion
		$this->assertFalse( wp_script_add_data( 'test-only-conditional', 'conditional', 'gt IE 7' ) );
```